src/primitives/transaction.h

```diff
@@ -375,6 +384,10 @@ struct CMutableTransaction
     std::vector<CTxOut> vout;
     uint32_t nLockTime;
     std::vector<CPourTx> vpour;
+    // TODO: This should be an unsigned char[33] (or boost array)
```

Comment:
Warning! That diff is out of date due to later commits in this PR. I recommend reviewing based on full changes.